prowler/providers/azure/services/sqlserver/sqlserver_vulnerability_assessment_enabled/sqlserver_vulnerability_assessment_enabled.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "azure",
+  "CheckID": "sqlserver_vulnerability_assessment_enabled",
+  "CheckTitle": "Ensure that Vulnerability Assessment (VA) is enabled on a SQL server by setting a Storage Account",
+  "CheckType": [],
+  "ServiceName": "sqlserver",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "medium",
+  "ResourceType": "SQLServer",
+  "Description": "Enable Vulnerability Assessment (VA) service scans for critical SQL servers and corresponding SQL databases.",
+  "Risk": "The Vulnerability Assessment service scans databases for known security vulnerabilities and highlights deviations from best practices, such as misconfigurations, excessive permissions, and unprotected sensitive data. Results of the scan include actionable steps to resolve each issue and provide customized remediation scripts where applicable. Additionally, an assessment report can be customized by setting an acceptable baseline for permission configurations, feature configurations, and database settings.",
+  "RelatedUrl": "https://docs.microsoft.com/en-us/azure/sql-database/sql-vulnerability-assessment",
+  "Remediation": {
+    "Code": {
+      "CLI": "Update-AzSqlServerVulnerabilityAssessmentSetting `-ResourceGroupName 'resource group name'` -ServerName 'Server Name'`-StorageAccountName 'Storage Name from same subscription and same Location' `-ScanResultsContainerName 'vulnerability-assessment' `-RecurringScansInterval Weekly `-EmailSubscriptionAdmins $true `-NotificationEmail @('mail1@mail.com' , 'mail2@mail.com')",
```

Comment:
```suggestion
      "CLI": "Update-AzSqlServerVulnerabilityAssessmentSetting -ResourceGroupName <resource group name> -ServerName <Server Name> -StorageAccountName <Storage Name from same subscription and same Location> -ScanResultsContainerName <vulnerability-assessment> -RecurringScansInterval Weekly -EmailSubscriptionAdmins $true -NotificationEmail @('mail1@mail.com' , 'mail2@mail.com')",
```

Reply:
Please review the syntax of the above command.

Reply:
Done!